src/rgw/rgw_reshard.cc

```diff
@@ -591,6 +596,10 @@ int RGWBucketReshard::do_reshard(int num_shards,
 	  return ret;
 	}
 
+  if (ret = f.check("abort_before_target_shard_entry"); ret < 0) { return ret; }
+
+  if (ret = f.check("error_before_target_shard_entry"); ret < 0) { return ret; }
+
```

Comment:
just need one check here for `"before_target_shard_entry"`, which will trigger on either `--inject-error-at=before_target_shard_entry` or `--inject-abort-at=before_target_shard_entry`

Reply:
yes. fixed